5.2.cup_of_join.py

```diff
@@ -0,0 +1,19 @@
+# The function accepts an unlimited number of lists and an option for an additional parameter,
+# and concatenates them into one list and the additional parameter between them
+def join(*lists, sep='-'):
+    if not lists:  # If no lists were received at all
+        return None
+    lst = [sep]
+    res = []
+    # Concatenate the lists into one list with the 'sep' parameter between them
+    for list1 in lists:
```

Comment:
Python is snake case -
https://www.google.com/amp/s/www.theserverside.com/answer/Camel-case-vs-snake-case-Whats-the-difference%3famp=1

Reply:
I fixed it.